static/app/utils/replays/hooks/useExtractedCrumbHtml.tsx

```diff
@@ -22,6 +22,7 @@ type Extraction = {
   crumb: Crumb;
   html: string;
   timestamp: number;
+  truncated: boolean;
```

Comment:
is something meant to read this value? it looks unused so far.

Reply:
> is there a screen shot showing a truncated html snippet?

Added a screenshot to the desc. You can see right now it just cuts off the snippet after x characters. I'm not doing anything with that value yet I wanted to see if anyone had suggestions about how these truncated snippets should be shown

Reply:
We can probably put that ticket in jesse's queue to think about. Right now it looks broken because the line just trails off and has no syntax highlighting.